bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
Note that it's not necessary to add `name:`  if the name is already included in the filename.